prow/oss/tests/jobs_test.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2019 Google LLC
```

Comment:
Wrong year?

Reply:
I just blindly copy/pasted this, but I can change it's year